### PR TITLE
New Package: tig

### PR DESCRIPTION
**Pull request by ledif**
This adds a package for [tig](https://jonas.github.io/tig/), which is an ncurses-based text-mode interface for git. It is a simple autotools project.

### CODE REVIEW

File: var/spack/repos/builtin/packages/tig/package.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#     spack install tig
+#
+# You can edit this file again by typing:
+#
+#     spack edit tig
+#
+# See the Spack documentation for more information on packaging.
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/tig/package.py]: Can you remove this template (everything below the copyright)?

Comment (ledif) [inline var/spack/repos/builtin/packages/tig/package.py]: Done.

File: var/spack/repos/builtin/packages/tig/package.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/tig/package.py]: You'll actually need 2 blank lines here to keep flake8 happy.

Comment (ledif) [inline var/spack/repos/builtin/packages/tig/package.py]: D'oh! Should be fixed now. :smile:

**Pull request merged**